data_summaries/Indic-Instruct.json

```diff
@@ -0,0 +1,483 @@
+      "Creative Writing",
+      "Instruction Following",
+      "Translation",
+      "Information Extraction"
+    ],
+    "Text Sources": [
+      "crowdsourced",
+      "wikipedia.org"
+    ],
+    "Model Generated": [
+      "IndicTrans2"
+    ],
+    "Format": [
+      "Zero-shot"
+    ],
+    "Human Annotation": "No",
+    "Derived from Datasets": [
+      "Databricks dolly"
+    ],
+    "Creators": [
+      "AI4Bharat",
+      "Indian Institute of Technology Madras",
+      "Indian Institute of Information Technology, Design and Manufacturing, Kancheepuram",
+      "Flipkart",
+      "University Of Surrey",
+      "Agency for Science, Technology and Research",
+      "National Institute of Information and Communication Technology",
+      "IBM",
+      "Microsoft"
+    ],
+    "Licenses": [
+      {
+        "License": "CC BY-SA 3.0",
+        "License URL": "https://huggingface.co/datasets/databricks/databricks-dolly-15k"
+      }
+    ],
+    "License Notes": "They utilized a variety of existing datasets, adopting diverse licenses in accordance with the licenses of the source datasets.",
+    "License Verified By": "Vipul",
+    "Dataset Filter IDs": [
+      "dolly"
+    ]
+  },
+  "indic-instruct-flan_v2": {
+    "Unique Dataset Identifier": "indic-instruct-flan_v2",
+    "Dataset Name": "indic-instruct-data-v0.1",
+    "Paper Title": "Airavata: Introducing Hindi Instruction-tuned LLM",
+    "Dataset URL": "https://huggingface.co/datasets/ai4bharat/indic-instruct-data-v0.1",
+    "GitHub URL": "https://github.com/AI4Bharat/IndicInstruct",
+    "Hugging Face URL": "https://huggingface.co/datasets/ai4bharat/indic-instruct-data-v0.1/viewer/flan_v2",
+    "Papers with Code URL": "",
+    "ArXiv URL": "https://arxiv.org/abs/2401.15006",
+    "Semantic Scholar Corpus ID": "267301288",
+    "Collection": "Indic-Instruct",
+    "Collection URL": "https://github.com/AI4Bharat/IndicInstruct",
+    "Languages": [
+      "English",
+      "Hindi"
+    ],
+    "Task Categories": [
+      "Instructional Question Answering",
+      "Closed-Book Question Answering",
+      "Open-Ended Question Answering",
+      "Natural Language Classification",
+      "Open-Domain Question Answering",
+      "Brainstorming",
+      "Summarization",
+      "Creative Writing",
+      "Translation",
+      "Information Extraction"
+    ],
+    "Text Sources": [],
+    "Model Generated": [
+      "IndicTrans2"
+    ],
+    "Format": [
+      "Zero-shot"
+    ],
+    "Human Annotation": "No",
+    "Derived from Datasets": [
+      "Flan v2"
+    ],
+    "Creators": [
+      "AI4Bharat",
+      "Indian Institute of Technology Madras",
+      "Indian Institute of Information Technology, Design and Manufacturing, Kancheepuram",
+      "Flipkart",
+      "University Of Surrey",
+      "Agency for Science, Technology and Research",
+      "National Institute of Information and Communication Technology",
+      "IBM",
+      "Microsoft"
+    ],
+    "Licenses": [
+      {
```

Comment:
Let's add a license: "Various" since Flan v2 has dozens of NC licenses within it.